mmwave/dataloader/adc.py

```diff
@@ -259,22 +259,68 @@ def _stop_stream(self):
         return self._send_command(CMD.RECORD_STOP_CMD_CODE)
 
     @staticmethod
-    def organize(raw_frame, num_chirps, num_rx, num_samples):
+    def organize(raw_frame, num_chirps, num_rx, num_samples, sensor_type='xWR14xx', real=False):
         """Reorganizes raw ADC data into a full frame
 
         Args:
             raw_frame (ndarray): Data to format
             num_chirps: Number of chirps included in the frame
             num_rx: Number of receivers used in the frame
             num_samples: Number of ADC samples included in each chirp
+            sensor_type: Type of sensor used for measurement, default is xWR14xx
 
         Returns:
             ndarray: Reformatted frame of raw data of shape (num_chirps, num_rx, num_samples)
 
         """
-        ret = np.zeros(len(raw_frame) // 2, dtype=complex)
+        ret = None
+
+        sensor_type = self.generalizedSensorType(sensor_type)
```

Comment:
i believe this is a wrong insertion

Reply:
What about it do you think is incorrect? I set ret to None b/c for real or complex data the size of ret should be different, they are then respectively defined in a later section. The generalizedSensorType is just one way of categorizing the different ways to partition the data.